CONTRIBUTING.md

```diff
@@ -76,12 +78,19 @@ Pull Requests will be automatically labled as "**Idle**" when:
 - No Engagement from the author of the PR to address Review Requests or input from the community for more than 15 calender days.
   - Personal holidays or public holidays will not be counted.
```

Comment:
I guess that can't be judge automatically ... label should be set anyway, someone (e.g. WSL) can delete in such exceptions.

Reply:
Yes I agree.